cylc/flow/tests/host_select_remote.py

```diff
@@ -0,0 +1,155 @@
+"""Test the cylc.flow.host_select module with remote hosts.
+
+NOTE: These tests require a remote host to work with and are skipped
```

Comment:
I think that you should offer a pointer on how to go about providing a remote host.

Reply:
Uses the same system as the functional tests (see `git grep 'remote host with shared fs'`).